tests/language/completion/pg_completion/test_smart_completion_public_schema_only.py

```diff
@@ -865,38 +865,40 @@ def test_set_schema(self, completer):
         self.assertEqual(result, expected)
 
     def test_keyword_lower_casing(self, ):
-        new_completer = PGCompleter(smart_completion=True, settings={'keyword_casing':'lower'})
-
+        new_completer = PGCompleter(smart_completion=True, settings={'keyword_casing': 'lower'})
         text = 'SEL'
         position = len(text)
+
+        # When I request completions from SEL with keyword_casing as 'upper'
         result = set(new_completer.get_completions(
             Document(text=text, cursor_position=position),
             Mock()))
 
-        # then completions should now be lower case
+        # Then completions should now be upper case
         self.assertSetEqual(result, set([Completion(text='select', start_position=-3, display_meta="keyword")]))
 
     def test_keyword_upper_casing(self, ):
-        new_completer = PGCompleter(smart_completion=True, settings={'keyword_casing':'upper'})
-
+        new_completer = PGCompleter(smart_completion=True, settings={'keyword_casing': 'upper'})
         text = 'sel'
         position = len(text)
+
+        # When I request completions from sel with keyword_casing as 'lower'
         result = set(new_completer.get_completions(
             Document(text=text, cursor_position=position),
             Mock()))
 
-        # then completions should now be lower case
+        # Then completions should now be lower case
         self.assertSetEqual(result, set([Completion(text='SELECT', start_position=-3, display_meta="keyword")]))
 
-    def test_keyword_auto_casing(self, ):        
-        new_completer = PGCompleter(smart_completion=True, settings={'keyword_casing':'auto'})
-
-        # if text is lower case   
+    def test_keyword_auto_casing(self, ):
+        new_completer = PGCompleter(smart_completion=True, settings={'keyword_casing': 'auto'})
         text = 'sel'
         position = len(text)
+
+        # When I request completions from sel with keyword_casing as 'auto'
```

Comment:
mention that as text is lower, the completion is lower as well. As it depends on actual text when it is auto